variant-box/pi-setup.sh

```diff
@@ -75,7 +75,65 @@ sudo zip -j   -u /boot/secrets.zip   /var/lib/tor/ssh/hostname
 
 
 # TODO: detect `-source` suffix in `bitcoind-version`; if present build from source, if not build from binaries
-# TODO: how to detect arch for binaries?
+BITCOINVER=`cat bundle/bitcoind-version`
+if [[ $BITCOINVER == *"source"* ]]; then 
+    echo "Building bitcoin from source"
+else
+    echo "Using pre-built binary"
```

Comment:
1. Why all these lines installing Docker instead of just `curl -fsSL get.docker.com | sh` ?
2. It might make sense to install Docker regardless of bitcoind, otherwise we'll need to duplicate docker-installation logic when it will come to LN node etc.

Reply:
Because it works, and I'd rather not fix whats not broken just yet.

If you think that it works then please do change it